### PR TITLE
Remove fullSync command

### PR DESCRIPTION
**Pull request by rmulhol**
- remove full sync command
- remove full sync mode from contract watcher
- remove code only used by either of the above

Thinking this code should be removed since we don't recommend using it and aren't actively maintaining it. Hoping that clearing out the clutter will make the codebase more nimble and assist with creating more helpful documentation.

- [X] remove "HeaderSync" qualifier in var/struct/migration names

### CODE REVIEW

**Comment by m0ar** [comment]
TODO still relevant or is this ready for review @rmulhol ?`

**Comment by rmulhol** [comment]
> TODO still relevant or is this ready for review @rmulhol ?`

Still relevant but I'll take care of it soon

File: db/migrations/00028_create_uncles_table.sql [hunk 1]
```diff
@@ -1,19 +0,0 @@
--- +goose Up
-CREATE TABLE public.uncles (
```
[the remark this answers is not in the document]
Comment (rmulhol) [reply db/migrations/00028_create_uncles_table.sql]: Yep!

File: integration_test/contract_test.go [hunk 2]
```diff
@@ -22,67 +22,17 @@ import (
...
-			transactionConverter := rpc2.NewRpcTransactionConverter(ethClient)
-			blockChain := eth.NewBlockChain(blockChainClient, rpcClient, node, transactionConverter)
-
-			logs, err := blockChain.GetFullSyncLogs(core.Contract{Hash: "0x123"}, big.NewInt(4703824), nil)
-
-			Expect(err).To(BeNil())
-			Expect(len(logs)).To(Equal(0))
-		})
-	})
-
```
[the remark this answers is not in the document]
Comment (rmulhol) [reply integration_test/contract_test.go]: Yep - removing `GetFullSyncLogs`

**Pull request merged**